Tests/NIOPosixTests/EventLoopTest.swift

```diff
@@ -2070,6 +2070,40 @@ final class EventLoopTest: XCTestCase {
         XCTAssertEqual("cool", actual)
     }
     #endif
+
+    func testRegressionSelectableEventLoopDeadlock() throws {
+        let iterations = 1_000
```

Comment:
note to reviewers: even `10` reproes basically 100% on my machine. 1000 should be really safe